dash-pipeline/bmv2/stages/outbound_routing.p4

```diff
@@ -33,8 +50,13 @@ control outbound_routing_stage(inout headers_t hdr,
             return;
         }
 
+        routing_group.apply();
+        if (meta.eni_data.routing_group_admin_state) {
         routing.apply();
+        } else {
+            drop(meta);
```

Comment:
if the packet can be dropped due to admin state down, it will be better to add a counter on top of routing group as stats.

Reply:
Done